materialize-redshift/driver.go

```diff
@@ -359,23 +385,22 @@ func (t *transactor) addBinding(
 
 	// Render templates that require specific S3 "COPY INTO" parameters.
 	for _, m := range []struct {
-		sql             *string
-		target          string
-		columns         []*sql.Column
-		truncateColumns bool
-		stagedFile      *stagedFile
+		sql        *string
+		target     string
+		columns    []*sql.Column
+		stagedFile *stagedFile
 	}{
-		{&b.copyIntoLoadTableSQL, fmt.Sprintf("flow_temp_table_%d", bindingIdx), target.KeyPtrs(), false, b.loadFile},
-		{&b.copyIntoMergeTableSQL, fmt.Sprintf("flow_temp_table_%d", bindingIdx), target.Columns(), true, b.storeFile},
-		{&b.copyIntoTargetTableSQL, target.Identifier, target.Columns(), true, b.storeFile},
+		{&b.copyIntoLoadTableSQL, fmt.Sprintf("flow_temp_table_%d", bindingIdx), target.KeyPtrs(), b.loadFile},
+		{&b.copyIntoMergeTableSQL, fmt.Sprintf("flow_temp_table_%d", bindingIdx), target.Columns(), b.storeFile},
+		{&b.copyIntoTargetTableSQL, target.Identifier, target.Columns(), b.storeFile},
 	} {
 		var sql strings.Builder
-		if err := tplCopyFromS3.Execute(&sql, copyFromS3Params{
-			Target:          m.target,
-			Columns:         m.columns,
-			ManifestURL:     m.stagedFile.fileURI(manifestFile),
-			Config:          t.cfg,
-			TruncateColumns: m.truncateColumns,
```

Comment:
We don't need this `TRUNCATECOLUMNS` option anymore since the runtime automatically keeps strings to less than 64k in length redshift materializations.